packages/api/src/platforms/vtex/clients/search/index.ts

```diff
@@ -120,6 +120,7 @@ export const IntelligentSearch = (
       sort,
       fuzzy,
       locale: ctx.storage.locale,
+      'show-invisible-items': 'true',
```

Comment:
Should this be always true? or should we open another PR with the possibility of set it using a config?